index.js

```diff
@@ -1,8 +1,17 @@
 module.exports = robot => {
   robot.on('issues.opened', async context => {
-    const options = context.repo({path: '.github/ISSUE_REPLY_TEMPLATE.md'});
-    const res = await context.github.repos.getContent(options);
-    const template = new Buffer(res.data.content, 'base64').toString();
+    let template;
+    try {
+      const options = context.repo({path: '.github/ISSUE_REPLY_TEMPLATE.md'});
+      const res = await context.github.repos.getContent(options);
+      template = new Buffer(res.data.content, 'base64').toString();
+    } catch (err) {
+      if (err.code === 404) {
+        template = 'We couldn\'t find an `ISSUE_REPLY_TEMPLATE` in your `.github` folder!';
+      } else {
+        throw err;
+      }
+    }
```

Comment:
The inability for apps to communicate with whoever set them up is an issue with all GitHub Apps right now. I'd love help thinking through some [best practices](https://github.com/probot/probot/blob/master/docs/best-practices.md) and documenting them.

In this case, I don't love commenting with the error because you can't be sure that the person commenting is the person that installed the integration, so it might be confusing.

I think I like the pattern I've seen from greenkeeper.io so far. It [opens a pull request](https://github.com/probot/probot/pull/103) when you first install it with any necessary changes, and will [open an issue](https://github.com/probot/probot.github.io/issues/1) something goes wrong.  The challenge is what to do when you install an app on an entire organization. Should it open PRs on all repos?

What do you think?